<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="menaka" failures="4" tests="4" name="Banking App Test(failed)(failed)" time="18.031" errors="0" timestamp="20 Jan 2022 03:36:59 GMT">
  <testcase classname="com.testCases.TC_LoginTest_001" name="loginTest" time="0.951">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:29)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.testCases.TC_LoginTest_002" name="loginDDT" time="4.14">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testCases.TC_LoginTest_002.loginDDT(TC_LoginTest_002.java:27)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.testCases.TC_LoginTest_002" name="loginDDT" time="3.898">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testCases.TC_LoginTest_002.loginDDT(TC_LoginTest_002.java:27)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.testCases.TC_LoginTest_002" name="loginDDT" time="3.968">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testCases.TC_LoginTest_002.loginDDT(TC_LoginTest_002.java:27)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- Banking App Test(failed)(failed) -->
